docs/docs/cmd/teams/message/message-reply-list.md

```diff
@@ -32,3 +32,66 @@ Retrieve the replies from a specified message from a channel of the Microsoft Te
 ```sh
 m365 teams message reply list --teamId 5f5d7b71-1161-44d8-bcc1-3da710eb4171 --channelId 19:88f7e66a8dfe42be92db19505ae912a8@thread.skype --messageId 1540747442203
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    [
+      {
+        "id": "1666799833162",
+        "replyToId": "1666799520731",
+        "etag": "1666799833162",
+        "messageType": "message",
+        "createdDateTime": "2022-10-26T15:57:13.162Z",
+        "lastModifiedDateTime": "2022-10-26T15:57:13.162Z",
+        "lastEditedDateTime": null,
+        "deletedDateTime": null,
+        "subject": null,
+        "summary": null,
+        "chatId": null,
+        "importance": "normal",
+        "locale": "en-us",
+        "webUrl": "https://teams.microsoft.com/l/message/19%3Ae2916df2b11046beba42d22da898383f%40thread.tacv2/1666799833162?groupId=aee5a2c9-b1df-45ac-9964-c708e760a045&tenantId=92e59666-257b-49c3-b1fa-1bae8107f6ba&createdTime=1666799833162&parentMessageId=1666799520731",
+        "policyViolation": null,
+        "eventDetail": null,
+        "from": {
+          "application": null,
+          "device": null,
+          "user": {
+            "id": "78ccf530-bbf0-47e4-aae6-da5f8c6fb142",
+            "displayName": "John Doe",
+            "userIdentityType": "aadUser",
+            "tenantId": "92e59666-257b-49c3-b1fa-1bae8107f6ba"
+          }
+        },
+        "body": {
+          "contentType": "text",
+          "content": "First reply"
+        },
+        "channelIdentity": {
+          "teamId": "aee5a2c9-b1df-45ac-9964-c708e760a045",
+          "channelId": "19:e2916df2b11046beba42d22da898383f@thread.tacv2"
```

Comment:
Let's try to align the output with the data used in the example. Same goes for output types text and csv.